.github/workflows/ci.yml

```diff
@@ -40,13 +40,17 @@ jobs:
         with:
           mill-version: ${{ env.MILL_VERSION }}
 
+      - name: Build packages
+        run: |
+          mill -i __.jar
+
```

Comment:
**thought:** seems to be the closest thing to `package` in sbt. Just compiling didn't discover the issues with missing traits because we need to run `resources`, at the very least.